### PR TITLE
Logo in form pages navigates to homepage

### PR DESCRIPTION
**Pull request by Ngoakor12**
Pull-Request for `paretOS`

## Description
- Added homepage link to logo in user creation related forms
- Fixed some typescript errors

## Relates to
- #306 

## Reviewers
- @person1 (merge duty)
- @person2
- ...

## Steps to reproduce (if describing bug fix)
- Start project
- Make request to `/endpoint` with `{ "body": "test" }`
- Expect some response

## Screenshots / other info
...

### CODE REVIEW

File: src/profile/LanguageSelector.tsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ const LanguageSelector = (props: { user: User }) => {
   };
 
   return (
-    <Box>
+    <Box component="div">
```
Comment (jeremydthomas) [inline src/profile/LanguageSelector.tsx:42]: everything works here, but Im just curious about this component=div, I redid this one here and I dont remember seeing anything about this. Can you point me to where you found it?

**Review by jeremydthomas — COMMENTED** [review]
Everything works and looks ok to me, just have that question where I asked it out of curiosity.

File: src/profile/LanguageSelector.tsx (hunk 1 above)
Comment (Ngoakor12) [reply src/profile/LanguageSelector.tsx:42]: I got a bunch of red lines when I opened that file...see image
image

I just looked up  the errror and found [this fix on stack overflow](https://stackoverflow.com/questions/68692230/ts-expression-produces-a-union-type-that-is-too-complex-to-represent-with-materi)

Comment (jeremydthomas) [reply src/profile/LanguageSelector.tsx:42]: Odd, but thank you for that, I think I can see why that would fix it.

**Review by jeremydthomas — APPROVED** [review]

**Pull request merged**